src/Collection/CollectionTrait.php

```diff
@@ -92,7 +92,7 @@ public function filter(?callable $callback = null): CollectionInterface
      */
     public function reject(callable $callback): CollectionInterface
     {
-        return new FilterIterator($this->unwrap(), fn($key, $value, $items) => !$callback($key, $value, $items));
+        return new FilterIterator($this->unwrap(), fn ($key, $value, $items) => !$callback($key, $value, $items));
```

Comment:
Looking at these change now I actually prefer not having a space after `fn`. I made this change in our CS rules since we require a space after `function` keyword too. Perhaps should change both to not require a space after the keyword in cakephp-codesniffer 5.x?

Reply:
I think the common standard for javascript is a space before the () in closures.